public/js/silder.js

```diff
@@ -0,0 +1,39 @@
+var slideIndex = 1;
+showSlides(slideIndex);
+var prev = document.querySelector('.prev');

```

Comment:
What if there are multiple sliders on the page, will this code still work correctly?

Reply:
Done